docs/docs/administer/running-sql-queries.md

```diff
@@ -20,6 +20,19 @@ used from the local shell like so:
   -c "SQL_STATEMENT"
 ```
 
+In addition, the `pg_dump` utility is also bundled; this can be used to take
```

Comment:
Thanks @gbe0! This looks good.

My only suggestion would be to add this as a section under the `Some examples of common tasks` list below.